docs/guides/add-custom-functionality/raw-sql.md

```diff
@@ -0,0 +1,36 @@
+# Raw SQL queries
+
+To run raw SQL queries using plugins, use `app.platformatic.db.query` method and passe to it a sql query using the `app.platformatic.sql` method.
+
+```js
+'use strict'
+module.exports = async(app, opts) => {
+  app.graphql.extendSchema(`
+    type YearlySales {
+      year: Int
+      sales: Int
+    }
+
+    extend type Query {
+      yearlySales: [YearlySales]
+    }
+  `)
+  app.graphql.defineResolvers({
+    Query: {
+      yearlySales: async(_, { title }) => {
+        const {sql} = app.platformatic;
+        const res = await app.platformatic.db.query(sql(`
+          SELECT
+            YEAR(created_at) AS year,
+            SUM(amount) AS sales
```

Comment:
would you mind adding a `where` clause here and use the `title` parameter?

Reply:
`where` to validate the userId ad an example?
for `title` I just forgot to remove after copy/past!